test.py

```diff
@@ -0,0 +1,22 @@
+from neo4j import GraphDatabase
+uri = "bolt://127.0.0.1:7687"
+driver = GraphDatabase.driver(uri, auth=("neo4j", "12341234"))
```

Comment:
Please, these variables should not be hardcoded. Create an env object within the process global object.